### ISSUE TITLE
Adjust long filename character limit

### ISSUE THREAD

**Issue by sjlevy** [wpsharks/comet-cache#888]
Suggest adjusting character limits formulated in issue #275 to be based on file system path length instead of URL, and to comply with the Windows path limitation (effectively 247 characters)

See post here for more details: https://wordpress.org/support/topic/long-filename-issues/#post-9066322

**Comment by jaswrks** [comment]
Just adding a related article posted on HN this morning.
https://eklitzke.org/path-max-is-tricky

**Comment by sjlevy** [comment]
Good point.  Maybe instead of a hard-coded length there could be an option to specify a max length in the Comet Cache settings?  That way each user could easily tailor it to their environment since there seem to be many factors.
